### ISSUE TITLE
Setting font-size in em breaks build

### ISSUE THREAD

**Issue by gyoshev** [telerik/kendo-themes#463]
## I'm submitting a...

* Regression report 

## Current behavior

Setting `$font-size: 1rem` breaks the compilation of the theme.

## Expected behavior

The theme can be compiled correctly if `$font-size: 1rem` is set.

## What is the motivation or use case for changing the behavior?

Allow customers to change the font-size of the theme so that components match their application styles.

## Environment
Package versions:

```
@progress/kendo-theme-default@2.59.0
```

## Cause and workaround

The problem is caused by the calculation of `$calendar-cell-size` that uses `$font-size` with the assumption that it is in pixels, following https://github.com/telerik/kendo-angular/issues/1891

To workaround the issue, set `$calendar-cell-size: 34px`.

**Comment by meriturva** [comment]
Still present on latest version kendo-theme-default: 3.1.0

**Comment by feitzi** [comment]
If I use `$calendar-cell-size: 34px.` i got a new build error:

```
  Incompatible units rem and px.
    ╷
350 │             top: calc( #{($padding-y-sm*2) + ($line-height*$font-size)} + 1px);
```
File: `\@progress\kendo-theme-default\scss\dataviz\_layout.scss (line 350, column 26)
`
See also telerik/kendo-themes#680

**Comment by feitzi** [comment]
this Issue is solved with #1060

**Comment by joneff** [comment]
#2565 fixes the issue completely as previously, using `rem` would have caused issues... We learned the lesson and will try not to have reverse engineered font-size based units.
